app/templates/buyers/award.html

```diff
@@ -44,14 +44,10 @@
                     {% endwith %}
 
                     {% block save_button %}
-                      {%
-                        with
-                        label="Save and continue",
-                        name="save_and_continue",
-                        type="save"
-                      %}
-                        {% include "toolkit/button.html" %}
-                      {% endwith %}
+                      {{ govukButton({
+                        "text": "Save and continue",
+                        "name": "save_and_continue",
```

Comment:
why do we need the name? Does the backend need to know which button was clicked?

Reply:
Again, I'm just going for a 1:1 conversion. It's possible the analytics looks for it, it's possible the unit tests look for it, it's possible the FTs look for it...

Reply:
Could we remove and see if any breaks? Otherwise, it will always exist and we will always wonder why it's being used.

Reply:
Could we remove and see if any breaks? Otherwise, it will always exist and we will always wonder why it's being used.

Reply:
Ok but we won't know for sure until it's got to the functional tests and even then I wouldn't be sure that e.g. our analytics code isn't trying to poke it... basically I'm afraid of scope creep here turning something that can be a straightforward task into something that requires much more thought and agreement - and we've got a long way to go on this task (I think this next round of apps are going to be significantly longer jobs than the previously converted frontends)

Reply:
I'd say leave it in - as Rob says it's the sort of thing that our event tracking analytics looks for.

Reply:
I'll not block this PR any further but I'm still a bit concerned that we have code which we are unsure of its purpose.

Reply:
Haha you wait until we get to the Supplier FE 😹

Reply:
> a bit concerned that we have code which we are unsure of its purpose

"welcome to digital marketplace"